### PR TITLE
Sort desc

### PR DESCRIPTION
**Pull request by eloiseespel**
Hi everyone! 

This is my pull request for sorting integer arrays in descending order in Java. 

It has 2 files: 
1. SortDesc.java
This file overrides the `compare()` method so that the user can use the `Arrays` library to sort in descending order.
2. SortDescTest.java. 
This file contains a simple unit test to showcase how to use SortDesc.java and for the user to make sure the code it working in the way intended. 

Please provide feedback on anything including stylistic, testing and comments/readability. 
Unit test is passing locally but feel free to pull my branch and test it on your own machine!

### CODE REVIEW

**Comment by Ansper11** [comment]
nice code

File: src/SortDesc.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import java.util.Comparator;
+
+public class SortDesc implements Comparator<Integer>{
+	@Override
+    public int compare(Integer o1, Integer o2) {
```
Comment (Ansper11) [inline src/SortDesc.java:12]: can you add some comments to explain how this function works?

**Comment by Nick-McGee** [comment]
Great use of built in Java features. Maybe add a comment to explain, even if it may seem obvious.

File: src/SortDescTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+class SortDescTest {
+
+	@Test
+	void test() {
+        int[] inArray = { 23, 76, 12, 43, 90, 30 };
+        int[] outArray = { 90, 76, 43, 30, 23, 12};
+        Arrays.sort(inArray);
```
Comment (yyht6300a) [inline src/SortDescTest.java:15]: It looks perfect!
The only thing I can suggest is to add some comments here.

**Review by Nick-McGee — APPROVED** [review]

**Review by Ansper11 — APPROVED** [review]

**Pull request merged**